### ISSUE TITLE
Segmentation results on ScanNet, discussion wanted!

### ISSUE THREAD

**Issue by Wuziyi616** [yanx27/Pointnet_Pointnet2_pytorch#108]
Hi! Thank you for your great open source work! I just wonder if you (or anyone else) has tried training this PointNet++ on ScanNet dataset in semantic segmentation task. I tried this with ScanNet dataset borrowed from other repos and got a **mIoU of ~50%** on the ScanNet **val set** and **mIoU ~45%** on the test set **with only XYZ input**. This result is much higher than their original submission, which achieves 33.9% as shown on the ScanNet online benchmark webpage.

Your network structure is slightly different from the official code but I don't think this can cause such a big improvement. Do you have any insight into this? The learning rate schedule maybe one cause, because I use cosine lr decay with 150 epochs from 1e-3 to 1e-5, and found this much better than their original step lr schedule. Can anyone share their results on ScanNet?

**Comment by yanx27** [comment]
@yanx27 Hi, results of semantic segmentation sometime depends on the protocol of data generation. What's repo did you use as your dataloader?

**Comment by Wuziyi616** [comment]
Thank you for your reply! I used the [pointconv](https://github.com/DylanWusee/pointconv) repo. BTW, have you tried training PN++ on ScanNet before?

**Comment by yanx27** [comment]
@Wuziyi616 I only tried PN++ in Tensorflow version, which only obtains similar results with official repo.

**Comment by Wuziyi616** [comment]
Yeah I understand. That's not a problem now. Thanks for your great open source work!

**Comment by MushroomLyn** [comment]
> Yeah I understand. That's not a problem now. Thanks for your great open source work!

Hello, may I ask how did you solve the problem that the mIoU of PN++ on the ScanNet dataset (~50%) is far lower than the official data of PN++ (~80%)? I also ran the code from this repository on ScanNet and obtained results similar to yours, with an mIoU of ~50%.
